diff --git a/node_modules/@rdfjs/parser-jsonld/lib/ParserStream.js b/node_modules/@rdfjs/parser-jsonld/lib/ParserStream.js
index fcfe40a..6ddc25c 100644
--- a/node_modules/@rdfjs/parser-jsonld/lib/ParserStream.js
+++ b/node_modules/@rdfjs/parser-jsonld/lib/ParserStream.js
@@ -63,11 +63,19 @@ class ParserStream {
       }
     })
 
-    parser.on('context', context => {
-      Object.entries(context).forEach(([prefix, iri]) => {
-        transform.emit('prefix', prefix, factory.namedNode(iri))
-      })
-    })
+    function processContext (context) {
+      if(Array.isArray(context)) {
+        return context.forEach(processContext)
+      }
+
+      if(typeof context === 'object') {
+        Object.entries(context).forEach(([prefix, iri]) => {
+          transform.emit('prefix', prefix, factory.namedNode(iri))
+        })
+      }
+    }
+
+    parser.on('context', processContext)
     parser.on('error', err => transform.destroy(err))
     parser.pipe(transform)
 
